SMLHelper/Initializer.cs

```diff
@@ -69,6 +71,8 @@ private static void Initialize()
             WorldEntityDatabasePatcher.Patch(harmony);
             IngameMenuPatcher.Patch(harmony);
             TooltipPatcher.Patch(harmony);
+            TechTypePatcher.cacheManager.SaveCache();
+            CraftTreeTypePatcher.cacheManager.SaveCache();
```

Comment:
Add a Debug level log before these new lines indicating that cache files are being saved